sig/OOP/school/library.rbs

```diff
@@ -0,0 +1,8 @@
+module OOP
+  module School
+    module Library
+      VERSION: String
+      # See the writing guide of rbs: https://github.com/ruby/rbs#guides
+    end
+  end
+end
```

Comment:
- [ ] Kindly remove all the unused files or ignore them by using git "gitignore" to keep your project clean and well organized.